scripts/debug_ws_full.py

```
@@ -72,25 +86,29 @@ async def test_config(header_name, headers, payload_name, payload):
-                WS_URL, 
-                **{ws_header_key: headers}, 
-                ping_interval=20, 
+                WS_URL,
+                **{ws_header_key: headers},
+                ping_interval=20,
                 ping_timeout=10,
                 ssl=ssl_context,
-                compression=None # Disable compression as per previous attempt
+                compression=None,  # Disable compression as per previous attempt
             ) as websocket:
                 logger.info(f"  [Connected] Status: 101")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)